phalcon/Cache.zep

```diff
@@ -60,6 +60,8 @@ class Cache implements CacheInterface
      */
     public function delete(var key) -> bool
     {
+        let key = (string) key;
```

Comment:
It is possible to change method signature -> `public function delete(string key)`

Reply:
No because PSR uses `mixed` for that.